contrib/babelfishpg_tsql/sql/upgrades/babelfishpg_tsql--5.1.0--5.2.0.sql

```diff
@@ -450,6 +450,25 @@ BEGIN
 END;
 $$;
 
+DO $$
+BEGIN
+IF NOT EXISTS(SELECT 1 FROM pg_class c JOIN pg_attribute a ON a.attrelid = c.oid WHERE c.relname = 'babelfish_partition_function' AND c.relnamespace::regnamespace::text = 'sys' AND a.attname = 'input_parameter_collation')
```

Comment:
update c.relname = 'babelfish_partition_function' to c.relname = 'babelfish_partition_function' COLLATE sys.DATABASE_DEFAULT, applies for other string comparison as well